tests/integration/conftest.py

```diff
@@ -170,9 +167,13 @@ def switch(self, name: str, path: Path):
             name (str):  Which application
             path (Path): Path to local charm
         """
-        app = self.applications[name]
-        app["charm"] = str(path.resolve())
-        app["channel"] = None
+        # FIXME: Omit non present charms
+        try:
+            app = self.applications[name]
+            app["charm"] = str(path.resolve())
+            app["channel"] = None
+        except KeyError:
+            log.warning("Application %s not found in bundle", name)
```

Comment:
whats this for? oh? for test bundles where theres no worker charm?

Reply:
Yes, it should be there until we add support for certificates in k8s-worker